includes/ting_search_autocomplete.ajax.inc

```diff
@@ -0,0 +1,128 @@
+<?php
```

Comment:
I'm not sure I understand your concern?... blank lines make for a more readable code, imho.